app/Http/Controllers/ProfileController.php

```diff
@@ -14,4 +17,31 @@ public function index()
    
         return view('profile', compact('posts'));
     }
+
+    public function postStatus(PostStatusRequest $request)
+    {
+        if ($request->hasFile('imageFile')) {
+            $post = Post::create([
+                'caption' => $request->caption,
+                'user_id' => Auth::id(),
+            ]);
+
+            foreach($request->file('imageFile') as $key => $file)
+            {
+                $name = rand().".". $file->getClientOriginalExtension();
```

Comment:
sao em lại nối chuỗi rỗng ntn?
`rand().".".`

Reply:
Dòng này của em dùng rand() để tạo tên ảnh bất kỳ. Còn hàm getExtension kia để lấy ra định dang của ảnh ví dụ như jpg hay png...
Em nối chuỗi bằng dấu chấm để tạo ra 1 tên ảnh bất kỳ với định dạng khi nó được chọn để upload ạ.

Reply:
```suggestion
                $name = rand() . "." . $file->getClientOriginalExtension();
```
em nên viết ntn để dễ hiều hơn nhé. sửa luôn nha

Reply:
Vâng ạ